### PR TITLE
Implement disabledFeatures in create API

### PR DESCRIPTION
**Pull request by SamuelWei**
Implements #149 and disabledFeaturesExclude https://github.com/bigbluebutton/bigbluebutton/pull/18005

### CODE REVIEW

**Comment by SamuelWei** [comment]
Also implement disabledFeaturesExclude https://github.com/bigbluebutton/bigbluebutton/pull/18005

File: src/Util/ParamsIterator.php [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+namespace BigBlueButton\Util;
+
+use PHPUnit\Framework\TestCase;
+
+/**
+ * @internal
+ *
+ * @coversNothing
+ */
+class ParamsIterator extends TestCase
```
Comment (FelixJacobi) [inline src/Util/ParamsIterator.php:30]: Same as in the other PR applies here.

**Review by FelixJacobi — CHANGES_REQUESTED** [review]

File: src/Util/ParamsIterator.php (hunk 1 above)
Comment (SamuelWei) [reply src/Util/ParamsIterator.php]: @FelixJacobi Updated according to the other PR, please re-review

File: tests/unit/Parameters/CreateMeetingParametersTest.php [hunk 2]
```diff
@@ -104,6 +107,32 @@ public function testCreateMeetingParameters()
         $this->assertEquals($newId, $createMeetingParams->getMeetingID());
     }
 
+    public function testDisabledFeatures()
```
Comment (FelixJacobi) [inline tests/unit/Parameters/CreateMeetingParametersTest.php:110]:
```suggestion
    public function testDisabledFeatures(): void
```

**Review by FelixJacobi — APPROVED** [review]

**Pull request merged**